tony-portal/app/views/tableMetadata.scala.html

```diff
@@ -1,12 +1,31 @@
 @(metricsList: Collection[JobMetadata])
 
-<html> <body>
+<html>
+
+  <head>
+
     <!-- Required meta tags for Bootstrap-->
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <!-- Bootstrap CSS -->
-  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css" integrity="sha384-GJzZqFGwb1QTTN6wy59ffF1BuGJpLSa9DkKMp0DgiMDm4iYMj70gZWKYbI706tWS" crossorigin="anonymous">
+  <meta charset="utf-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
+
+    <!-- Bootstrap core CSS -->
+    <link rel="stylesheet" media="screen" href='@routes.Assets.versioned("portal_assets/bootstrap-4.3.1-dist/css/bootstrap.min.css")'>
+
+  <style>
+      .rmv_elmt {
+        color: rgba(0, 0, 0, 0)
+      }
 
+      body {
+        margin:12px;
+      }
+
+  </style>
+
+</head>
```

Comment:
Should the <style> section and the </head> tag all be indented two more spaces?

Reply:
@erwa reformatted code, please take a look